### PR TITLE
docs: remove version number text from docs

### PR DESCRIPTION
**Pull request by sangshuduo**

### CODE REVIEW

File: docs/en/20-third-party/06-statsd.md [hunk 1]
```diff
@@ -39,16 +39,9 @@ $ echo "foo:1|c" | nc -u -w0 127.0.0.1 8125
 Use the TDengine CLI to verify that StatsD data is written to TDengine and can read out correctly.
 
 ```
-Welcome to the TDengine shell from Linux, Client Version:2.4.0.0
+Welcome to the TDengine shell from Linux, Client Version:3.0.0.0
 Copyright (c) 2020 by TAOS Data, Inc. All rights reserved.
```
Comment (ericyangpan) [inline docs/en/20-third-party/06-statsd.md]: Copyright (c) 2022

File: docs/zh/14-reference/03-connector/python.mdx [hunk 2]
```diff
@@ -153,7 +153,7 @@ curl -u root:taosdata http://<FQDN>:<PORT>/rest/sql -d "select server_version()"
   "status": "succ",
   "head": ["server_version()"],
   "column_meta": [["server_version()", 8, 8]],
-  "data": [["2.4.0.16"]],
+  "data": [["3.0.0.0"]],
```
Comment (ericyangpan) [inline docs/zh/14-reference/03-connector/python.mdx]: Result schema should be 3.0 style.

File: docs/en/25-application/02-collectd.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Please refer to the [official documentation](https://grafana.com/grafana/downloa
 
 ### Install TDengine
 
-Download the latest TDengine-server 2.4.0.x or above from the [Downloads](http://taosdata.com/cn/all-downloads/) page on the TAOSData website and install it.
+Download the latest TDengine-server from the [Downloads](http://taosdata.com/cn/all-downloads/) page on the TAOSData website and install it.
```
Comment (ericyangpan) [inline docs/en/25-application/02-collectd.md]: Use tdengine.com download url in English docs?
But https://tdengine.com/all-downloads/ is still 2.6 versions.

@gccgdb1234

File: docs/en/25-application/01-telegraf.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ Please refer to the [official documentation](https://grafana.com/grafana/downloa
 
 ### TDengine
 
-Download the latest TDengine-server 2.4.0.x or above from the [Downloads](http://taosdata.com/cn/all-downloads/) page on the TAOSData website and install it.
+Download the latest TDengine-server from the [Downloads](http://taosdata.com/cn/all-downloads/) page on the TAOSData website and install it.
```
Comment (ericyangpan) [inline docs/en/25-application/01-telegraf.md]: Use tdengine.com download url in English docs?

**Review by arielyang — CHANGES_REQUESTED** [review]
Totally, seems need https://tdengine.com/all-downloads/ page.

**Pull request merged**